py3status/modules/getjson.py

```diff
@@ -94,7 +94,7 @@ def _query_url(self):
             resp = self.py3.request(self.url, timeout=self.timeout)
             status = resp.status_code == 200
             resp = resp.json()
-        except self.Py3.RequestException:
+        except self.py3.RequestException:
```

Comment:
I'll rebase this branch on master to do a final test when it will seem like we are close to merging it, and this will automatically be resolved by git 😉